StableIPFSLoader.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="fragment" content="!">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Decentralized Prediction Markets | Augur</title>
+  <meta name="description"
+        content="Augur is a decentralized oracle and prediction market protocol built on the Ethereum blockchain. It allows you to forecast events and be rewarded for predicting them correctly.">
+  <script src="https://unpkg.com/ipfs/dist/index.min.js"></script>
+  <script>
+      (async function() {
+          const node = await window.Ipfs.create();
+          const pageData = await fetch(window.location.href);
+
+          if (!pageData.ok) {
+              return console.error(pageData.statusText);
+          } else {
+              await node.add(await pageData.text(), (error, response) => {
+                  if (error) {
+                      console.error(error);
+                  } else {
+                      console.log('IPFS response:', response);
+                  }
+              });
+          }
+
+          let githubData, ipfsHash;
+
+          try {
+              const githubReleaseApiUrl = 'https://api.github.com/repos/AugurProject/augur/releases';
+              githubData = await fetch(githubReleaseApiUrl);
+
+              if (!githubData.ok) {
+                  return console.error(data.statusText);
+              }
+
+              githubData = await githubData.json();
+
+              if (githubData.length === 0) {
+                  return console.error('It appears that Augur has no releases.');
```

Comment:
Same here with an error message. Once we get this into testing @petervecchiarelli can work on the messaging.